### PR TITLE
adaptive simulcast change log update

### PR DESCRIPTION
**Pull request by makarandp0**
Fixed couple of typos, and added additional notes about limitation of the feature.
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 New Features
 ------------
 
-This release introduces new beta feature **Adaptive Simulcast**. This opt-in feature can be enabled by setting `preferredVideoCodes="auto"` in connect options. When joining a group room with this feature enabled SDK will use VP8 simulcast, and will enable/disable simulcast layers dynamically, thus improving bandwidth and CPU usage. It works best when used along with `Client Track Switch Off Controls` and `Video Content Preferences`. These two flags allows SFU to determine which simulcast layers are needed, thus allowing it to disable the layers not needed on publisher side.
+This release introduces new beta feature **Adaptive Simulcast**. This opt-in feature can be enabled by setting `preferredVideoCodecs="auto"` in connect options. When joining a group room with this feature enabled SDK will use VP8 simulcast, and will enable/disable simulcast layers dynamically, thus improving bandwidth and CPU usage. It works best when used along with `Client Track Switch Off Controls` and `Video Content Preferences`. These two flags allows SFU to determine which simulcast layers are needed, thus allowing it to disable the layers not needed on publisher side.
```
Comment (anthonylazaro) [inline CHANGELOG.md:10]:
```suggestion
This release introduces new beta feature **Adaptive Simulcast**. This opt-in feature can be enabled by setting `preferredVideoCodecs="auto"` in connect options. When joining a group room with this feature enabled the SDK will use VP8 simulcast, and will enable/disable simulcast layers dynamically, thus improving bandwidth and CPU usage. It works best when used along with `Client Track Switch Off Controls` and `Video Content Preferences`. These two flags allows the SFU to determine which simulcast layers are needed, thus allowing it to disable the layers not needed on publisher side.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,7 +27,14 @@ Example:
   });
   ```
 
-Note that this option is incompatible with connect option `maxVideoMaxBitrate`, and will result in an error at connect time if specified along with the new options.
+Additional notes:
```
Comment (anthonylazaro) [inline CHANGELOG.md:30]:
```suggestion
Known Limitations:
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -27,7 +27,14 @@ Example:
...
+- This feature is incompatible with connect option `maxVideoBitrate`, and will result in an error at connect time if specified along with the new option.
```
Comment (anthonylazaro) [inline CHANGELOG.md:31]: I think this should be moved up higher so this section can be focused on known limitations. This is more instructions for use. I would add the following known limitation:

"The beta currently does not support setting a max bitrate for your simulcast layers."

File: CHANGELOG.md [hunk 4]
```diff
@@ -27,7 +27,14 @@ Example:
...
+- This feature is not supported on some configurations. In following cases specifying `preferredVideoCodecs="auto"` will revert back to using unicast.
+  - Firefox,
+  - When room is configured to use H264 codec
+  - Peer-to-Peer rooms.
+- When room is being recorded, SFU will not disable layers, as they need to be recorded.
```
Comment (anthonylazaro) [inline CHANGELOG.md:36]:
```suggestion
- When the room or participant publishing is being recorded, the SFU will not disable any simulcast layers.
```

**Review by anthonylazaro — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -27,7 +27,15 @@ Example:
   });
   ```
 
-Note that this option is incompatible with connect option `maxVideoMaxBitrate`, and will result in an error at connect time if specified along with the new options.
+Known Limitations
+-----------------
+
+- This feature is not supported on some configurations. In following cases specifying `preferredVideoCodecs="auto"` will revert back to using unicast.
```
Comment (anthonylazaro) [inline CHANGELOG.md:33]:
```suggestion
- This feature is not supported on some configurations. In the following cases specifying `preferredVideoCodecs="auto"` will revert back to using unicast.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -27,7 +27,15 @@ Example:
...
+  - When publisher is using firefox
+  - When room is configured to use H264 codec
```
Comment (anthonylazaro) [inline CHANGELOG.md:35]:
```suggestion
  - When room is configured to use H.264 codec
```

**Review by anthonylazaro — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 7]
```diff
@@ -7,7 +7,7 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
...
+This release introduces new beta feature **Adaptive Simulcast**. This opt-in feature can be enabled by setting `preferredVideoCodecs="auto"` in connect options. When joining a group room with this feature enabled the SDK will use VP8 simulcast, and will enable/disable simulcast layers dynamically, thus improving bandwidth and CPU usage. It works best when used along with `Client Track Switch Off Controls` and `Video Content Preferences`. These two flags allows the SFU to determine which simulcast layers are needed, thus allowing it to disable the layers not needed on publisher side. The beta currently does not support setting a max bitrate for your simulcast layers.
```
Comment (anthonylazaro) [inline CHANGELOG.md:10]:
```suggestion
This release introduces a new beta feature **Adaptive Simulcast**. This opt-in feature can be enabled by setting `preferredVideoCodecs="auto"` in connect options. When joining a group room with this feature enabled the SDK will use VP8 simulcast, and will enable/disable simulcast layers dynamically, thus improving bandwidth and CPU usage. It works best when used along with `Client Track Switch Off Controls` and `Video Content Preferences`. These two flags allows the SFU to determine which simulcast layers are needed, thus allowing it to disable the layers not needed on publisher side. The beta currently does not support setting a max bitrate for your simulcast layers.
```

**Review by anthonylazaro — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 8]
```diff
@@ -7,7 +7,7 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
...
+This release introduces a new beta feature **Adaptive Simulcast**. This opt-in feature can be enabled by setting `preferredVideoCodecs="auto"` in connect options. When joining a group room with this feature enabled the SDK will use VP8 simulcast, and will enable/disable simulcast layers dynamically, thus improving bandwidth and CPU usage. It works best when used along with `Client Track Switch Off Controls` and `Video Content Preferences`. These two flags allows the SFU to determine which simulcast layers are needed, thus allowing it to disable the layers not needed on publisher side. The beta currently does not support setting a max bitrate for your simulcast layers.
```
Comment (manjeshbhargav) [inline CHANGELOG.md:10]:
```suggestion
This release introduces a new beta feature **Adaptive Simulcast**. This opt-in feature can be enabled by setting `preferredVideoCodecs="auto"` in ConnectOptions. When joining a group room with this feature enabled, the SDK will use VP8 simulcast, and will enable/disable simulcast layers dynamically, thus improving bandwidth and CPU usage. It works best when used along with `Client Track Switch Off Controls` and `Video Content Preferences`. These two flags allows the SFU to determine which simulcast layers are needed, thus allowing it to disable the layers not needed on publisher side. The beta currently does not support setting a max bitrate for your simulcast layers.
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -27,7 +27,15 @@ Example:
...
 
-Note that this option is incompatible with connect option `maxVideoMaxBitrate`, and will result in an error at connect time if specified along with the new options.
+Known Limitations
+-----------------
+
+- This feature is not supported on some configurations. In the following cases specifying `preferredVideoCodecs="auto"` will revert back to using unicast.
+  - When publisher is using firefox
+  - When room is configured to use H.264 codec
+  - Peer-to-Peer rooms
+- When the room or participant publishing is being recorded, the SFU will not disable any simulcast layers.
```
Comment (manjeshbhargav) [inline CHANGELOG.md:37]:
```suggestion
- Specifying `preferredVideoCodecs="auto"` will revert to unicast in the following cases:
  - The publisher is using Firefox.
  - The publisher has preferred the H264 codec.
  - The Room is configured to support only the H264 codec.
  - Peer-to-Peer Rooms
- When the Room is being recorded, the SFU will not disable any simulcast layers of the publisher's VideoTrack.
```

**Review by manjeshbhargav — APPROVED** [review]

**Pull request merged**